src/BoundaryConditions/fill_halo_regions_zipper.jl

```diff
@@ -93,7 +93,6 @@ end
         end
     end
 
-    @inbounds v[i, Ny, k] = ifelse(i > Nx ÷ 2, sign * v[i′, Ny + 1, k], v[i, Ny, k])
```

Comment:
@simone-silvestri I think we need this line. Did you remove it on purpose?

Reply:
Yes, why do we need this line? The v velocity is not duplicated across the seam ad the tracer and u-velocity is

Reply:
Not sure I understand. @NoraLoose's sketch showed that `v` is duplicated. What is the additional information that negates it?

Reply:
Oh "evolved", not "involved"

Reply:
>  So therefore we have the opposite rule for v than for other fields, and we should be filling in the reverse direction (eg putting interior values from the right of the seam onto the left boundary).

I think we are always taking values from the ride side, and copying them over to the left side (possibly with a sign change).

Reply:
Basically, we compute tendencies only up to Ny, so the Ny+1 must be determined by filling halos